ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -2778,9 +2778,19 @@ namespace
 Pointer<Database>
 setup_fe_projector_db(const Pointer<Database>& input_db)
 {
-    Pointer<Database> db(new InputDatabase(input_db->getName() + "::FEProjector"));
-    if (input_db->keyExists("num_fischer_vectors"))
-        db->putInteger("num_fischer_vectors", input_db->getInteger("num_fischer_vectors"));
+    Pointer<Database> db;
+    if (input_db->keyExists("FEProjector"))
+    {
+        db = input_db->getDatabase("FEProjector");
+    }
+    else
+    {
+        db = new InputDatabase("FEProjector");
+        if (input_db->keyExists("num_fischer_vectors"))
+        {
+            db->putInteger("num_fischer_vectors", input_db->getInteger("num_fischer_vectors"));
+        }
+    }
```

Comment:
This type of refactoring really should be in another PR but lets work with what we have.